src/server/scripts/Northrend/IcecrownCitadel/boss_the_lich_king.cpp

```diff
@@ -765,6 +765,13 @@ class boss_the_lich_king : public CreatureScript
             }
         }
 
+        uint32 CalculateHealthPercentageAfterHit(uint32 damage)
```

Comment:
This method already exists in Unit, e.g. `me->HealthBelowPctDamaged(70, damage)`

https://github.com/azerothcore/azerothcore-wotlk/blob/0f5992045ccd8e7c9fc320cd21f2cd4184552de4/src/server/game/Entities/Unit/Unit.h#L1539

Reply:
cool didnt now that,